audiorecorder/src/test/java/org/odk/collect/audiorecorder/mediarecorder/AMRRecordingResourceTest.kt

```diff
@@ -0,0 +1,29 @@
+package org.odk.collect.audiorecorder.mediarecorder
+
+import android.media.MediaRecorder
+import org.junit.Test
+import org.mockito.Mockito.inOrder
+import org.mockito.Mockito.mock
+
+class AMRRecordingResourceTest {
+
+    /**
+     * Calls to stop() while paused hang which appears to be a problem in the Android
+     * framework. Resuming and the stopping does work however.
+     *
+     * @see <a href="https://gist.github.com/seadowg/e6cbab21e032fe2fcd9afb9b4f458ab9">Gist</a> for
+     * test demonstrating the issue in Android.
+     */
+    @Test
+    fun whenPaused_stop_callsResumeThenStopOnMediaRecorder() {
```

Comment:
Don't we need another test but for lower android api?

Reply:
Do you mean for when `resume` isn't supported (below API 24)? I didn't need a test to drive out the change but I think you're right that it's important to check we don't call `MediaRecorder#resume` directly here - that would explode! I'll add the test.

Reply:
Yes that what I meant.